integrations/kosli/manifest.json

```diff
@@ -38,10 +38,14 @@
         "method": "POST",
         "headers": [
           {
-            "name": "x-ld-signature",
-            "value": "{{secret}}"
+            "name": "Content-Type",
+            "value": "application/json"
           }
-        ]
+        ],
+        "hmacSignature": {
+          "headerName": "X-LD-Signature",
+          "hmacSecretFormVariableKey": "secret"
+        }
```

Comment:
Is the intent here to switch to a different form of authentication for the integration? 
If that is the case, and you would want to start signing messages with an HMAC signature, further code changes would be required

Reply:
Yes the intention was always to use hmac validation. The manifest was not implemented correctly, this is a fix.